fe/fe-core/src/main/java/com/starrocks/planner/LookUpNode.java

```diff
@@ -0,0 +1,65 @@
+// Copyright 2021-present StarRocks, Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.starrocks.planner;
+
+import com.starrocks.analysis.RowPositionDescriptor;
+import com.starrocks.thrift.TExplainLevel;
+import com.starrocks.thrift.TLookUpNode;
+import com.starrocks.thrift.TPlanNode;
+import com.starrocks.thrift.TPlanNodeType;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public class LookUpNode extends PlanNode {
```

Comment:
1. How about add GLM `prefix` for `LookUp/Fetch` Node to ensure those nodes are used for `glm`?

2. is LookUpNode only used for `glm`, can we extend it for `index seek` or `index scan` later?